draft-ietf-netmod-schema-mount.org

```diff
@@ -239,9 +225,14 @@ container. Data in this container is intended to be as stable as data
 in the top-level YANG library ^RFC7895^. In particular, it SHOULD NOT
 change during the same management session.
 
-The modules that are mounted under a mount point has no relation to
+Generally, the modules that are mounted under a mount point has no
+relation to
 the modules in the parent schema; specifically, if a module is mounted
-it may or may not be present in the parent schema.
+it may or may not be present in the parent schema and, if present, its
+data may have no relationship to the data contained in the parent
+module.  Exceptions are possible, and such needs to be defined in the
+model defining the exception, e.g., YANG library data as noted below
+and the interface module in ^I-D.ietf-rtgwg-lne-model^. 
```

Comment:
I don't think the YANG library data is an exception - the data in the mounted YANG library has no relation to the parent's YANG library contents.